macspecial.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
```

Comment:
Not sure if this is needed if compiled with -std=c++11

Reply:
-std=c++11 is not working under mac, with this flagg lots of errors are generated:

     cmf/cmf_core_src/atmosphere/../math/timeseries.h:82:17: error: no type named 
    'shared_ptr' in namespace 'std'